docs/imagecustomizer/api/configuration/inputBootloader.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Configuration
+---
+
+# inputBootloader type
+
+Specifies the configuration for inputting signed bootloader components, including shim and systemd-boot, into the image.
+
+Example:
+
+```yaml
+bootloader:
+  path: /input/bootloader
```

Comment:
Should we keep this as a folder with 'expected contents' or break down the field into individual field for each of the individual components?